dlib/cmake_utils/test_for_cudnn/find_cudnn.txt

```diff
@@ -4,7 +4,7 @@ message(STATUS "Looking for cuDNN install...")
 # libraries and also a few other places as well.
 find_path(cudnn_include cudnn.h
     HINTS ${CUDA_INCLUDE_DIRS} ENV CUDNN_INCLUDE_DIR  ENV CUDNN_HOME
-    PATHS /usr/local ENV CPATH
+    PATHS /usr/local/cuda ENV CPATH
```

Comment:
On most systems it is in /usr/local, not /usr/local/cuda.  So this will break things for most people.  Does listing both paths work for you?  E.g. as is done immediately below.

Reply:
Yes it did work  and i have updated the pull request . please have a look